clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -96,6 +96,15 @@ def err_drv_cuda_host_arch : Error<
 def err_drv_no_sycl_libspirv : Error<
   "cannot find '%0'; provide path to libspirv library via '-fsycl-libspirv-path', or "
   "pass '-fno-sycl-libspirv' to build without linking with libspirv">;
+def warn_flag_no_sycl_libspirv
+    : Warning<"no-sycl-libspirv option is not meant for regular application "
+              "development and severely limits the correct behavior of DPC++ "
+              "when using the '%0' triple">,
+      InGroup<NoLibspirvHipCuda>;
+def warn_flag_no_sycl_libspirv_has_no_effect
+    : Warning<"no-sycl-libspirv option has no effect when compiled with the "
```

Comment:
How about - `ignoring option 'fno-sycl-libspirv' for triple "%0"`

Reply:
Maybe generalize even more for possible usage in the future: `ignoring option '%0' for triple '%1'`.
We have an existing `warn_drv_unsupported_option_for_target` but the verbiage may not match what we want.

Reply:
Yeah makes sense to me. I'll make this change.